packages/block-editor/src/components/block-draggable/index.native.js

```diff
@@ -210,7 +220,14 @@ const BlockDraggableWrapper = ( { children } ) => {
 				style={ chipStyles }
 				pointerEvents="none"
 			>
-				<DraggableChip />
+				{ draggedBlockIcon && (
+					<Animated.View
+						entering={ ZoomInEasyDown.duration( 200 ) }
+						exiting={ ZoomOutEasyDown.duration( 150 ) }
```

Comment:
These duration values have been taken from [this comment](https://github.com/wordpress-mobile/gutenberg-mobile/pull/4691#issuecomment-1101872877) regarding design feedback.